ESP_ATMod/command.cpp

```diff
@@ -1643,6 +1669,165 @@ void cmd_AT_CIPMUX()
 	}
 }
 
+/*
+ * AT+CIPCIPSERVER - Deletes/Creates TCP Server
+ */
+void cmd_AT_CIPSERVER()
+{
+
+	if (!gsCipMux)
+	{
+		Serial.printf_P(MSG_ERROR);
+		return;
+	}
+	uint8_t error = 1; // 1 = generic error, 0 = ok
+	uint16_t offset = strlen("AT+CIPSERVER");
+
+	bool stop = false;
+	uint32_t port = 0;
+
+	do
+	{
+		if (inputBuffer[offset] != '=')
+			break;
+
+		++offset;
+		stop = inputBuffer[offset] == '0';

```

Comment:
As far as I understand, the first parameter has 2 allowed values only: `0` and `1`. I'd prefer checking the parameter fully, i.e. throw an error when it is not either 0 or 1.